### PR TITLE
Improve performance of ControllerInstanceToggleTest

### PR DESCRIPTION
**Pull request by jfim**
Improve the execution time of ControllerInstanceToggleTest by
removing numerous sleep statements.

### CODE REVIEW

**Comment by jfim** [comment]
From testing on my desktop (before and after):
```
testInstanceToggle(com.linkedin.pinot.controller.helix.ControllerInstanceToggleTest)  Time elapsed: 226.345 sec
testInstanceToggle(com.linkedin.pinot.controller.helix.ControllerInstanceToggleTest)  Time elapsed: 46.521 sec
```

File: pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/ControllerInstanceToggleTest.java [hunk 1]
```diff
@@ -87,9 +106,15 @@ public void testInstanceToggle() throws Exception {
...
+      pollForEqualityAssertion(new Function<Object, Object>() {
+        @Nullable
+        @Override
+        public Object apply(@Nullable Object input) {
+          return _helixAdmin.getResourceIdealState(HELIX_CLUSTER_NAME, tableName + "_OFFLINE").getNumPartitions();
+        }
+      }, expectedSegmentCount, 2000, 10);
     }
 
     Thread.sleep(2000);
```
Comment (Jackie-Jiang) [inline pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/ControllerInstanceToggleTest.java]: What is this sleep for?

Comment (jfim) [inline pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/ControllerInstanceToggleTest.java]: You can ask `xiafu` :)

**Comment by jfim** [comment]
It's not done for delete because there's only one deletion. In this test, there are 20 broker and server instances that get turned on and off as well as 10 segments, for a total of 90 pauses of 2 seconds.

**Comment by npawar** [comment]
Closed this PR due to 6 months inactivity. Reopen if needed.

**Pull request closed without merging**